modules/wizard/wizard.php

```diff
@@ -183,6 +183,13 @@ public function setup_wizard_page() {
 			exit;
 		}
 
+		$home_page_id = get_option( 'page_on_front' );
+		$home_page_language = $this->model->post->get_language( $home_page_id );
+
+		if ( ! $home_page_language && 'home-page' === $this->step ) {
+			wp_safe_redirect( esc_url_raw( $this->get_step_link( 'untranslated-contents' ) ) );
+			exit;
+		}
```

Comment:
Shouldn't we protect all steps after the content step as we did for the languages? This would protect the WooCommerce step too.

Reply:
Done.